src/app/global/local_account_sensitive_settings.nim

```diff
@@ -586,6 +588,20 @@ QtObject:
     write = setGifUnfurlingEnabled
     notify = gifUnfurlingEnabledChanged
 
+  proc firstCommunityCreatedChanged(self: LocalAccountSensitiveSettings) {.signal.}
+  proc getFirstCommunityCreated(self: LocalAccountSensitiveSettings): bool {.slot.} =
+    self.settings.value(LSS_KEY_COMMUNITY_CREATED, newQVariant(DEFAULT_LSS_KEY_COMMUNITY_CREATED)).boolVal
+  proc setFirstCommunityCreated(self: LocalAccountSensitiveSettings, value: bool) {.slot.} =
+    if value == self.getFirstCommunityCreated:
```

Comment:
Since the logic is just about seeing the create community popup, maybe we should rename it `createCommunityPopupSeen`?

Reply:
Fixed